src/components/Signup/FormField.js

```diff
@@ -1,16 +1,15 @@
 import React, { Component, PropTypes } from 'react';
+import '../../styles/Signup.scss';
 
 class FormField extends Component {
```

Comment:
I think the component `FormField` is too generic to be in `Signup` folder. You can reuse this component in different scenarios. I suggest create a folder `Common` for all the components that are generic